packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -563,6 +568,10 @@ function ContainersMenuSection(props: ContainersMenuSectionProps): React.ReactEl
 						text={containerKey}
 						onClick={(event): void => {
 							selectContainer(`${containerKey}`);
+							usageLogger?.sendTelemetryEvent({
+								eventName: "ContainerClicked",
+								containerKey,
```

Comment:
Let's remove the `containerKey`, per the standup discussion today. It's technically content that the user gives us, so we should not have it in the extension's usage telemetry.